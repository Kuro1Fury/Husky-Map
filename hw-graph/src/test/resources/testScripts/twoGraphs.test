# twoGraphs test
# Test that two graphs can be created at the same time

# Create graphs
CreateGraph graph1
CreateGraph graph2

