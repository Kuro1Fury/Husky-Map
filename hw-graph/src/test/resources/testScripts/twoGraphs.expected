# twoGraphs test
# Test that two graphs can be created at the same time

# Create graphs
created graph graph1
created graph graph2
